doc/manual/install-language.rst

```diff
@@ -0,0 +1,97 @@
+Appendix: Installing the example languages
+==========================================
+
+DOMjudge ships with some default languages with a default configuration.
+As you might setup contests with those languages we provide how those languages were
```

Comment:
Words like "setup" are written together when used as noun, but separate when used as verbs.
```suggestion
As you might set up contests with those languages we provide how those languages were
```

Reply:
The text doesn't read so smoothly, but just merge after this change. I'll see if I can go through the whole documentation some time and improve things.